src/http-proxy/src/lib.rs

```diff
@@ -1,19 +1,53 @@
 // Copyright Materialize, Inc. and contributors. All rights reserved.
 //
-// Use of this software is governed by the Business Source License
-// included in the LICENSE file.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License in the LICENSE file at the
+// root of this repository, or online at
 //
-// As of the Change Date specified in that file, in accordance with
-// the Business Source License, use of this software will be governed
-// by the Apache License, Version 2.0.
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 #![deny(missing_docs)]
+#![cfg_attr(nightly_doc_features, feature(doc_cfg))]
 
-//! HTTP utilities.
+//! [<img src="https://materialize.com/wp-content/uploads/2020/01/materialize_logo_primary.png" width=180 align=right>](https://materialize.com)
+//!
+//! HTTP proxy adapters.
 //!
 //! This crate constructs HTTP clients that respect the system's proxy
 //! configuration.
 //!
+//! # Maintainership
+//!
+//! This crate is developed as part of [Materialize], the streaming data
+//! warehouse. Contributions are encouraged:
```

Comment:
Might want to check with Craig but I think "the streaming data warehouse" branding might be deprecated.

Reply:
🤷🏽